CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# dbt_servicenow v0.3.0
+[PR #10](https://github.com/fivetran/dbt_servicenow/pull/10) includes the following updates:
+
+## 🚨 Breaking Changes
+-  The following fields were removed from the `servicenow__task_enhanced` model as `problem_first_reported_by_task_value` was incorrectly assumed to have corresponding `sys_user` values, but in actuality is related to the task table and therefore have no user fields.
+  - `problem_reporter_email, problem_reporter_manager_value, problem_reporter_department_value, problem_reporter_name, problem_reporter_roles`
```

Comment:
Can you make these their own bullets to make it clear what fields were removed instead of adding them all in a single code block. For example, I'm thinking of the following format:
```suggestion
  - `problem_reporter_email`
  - `problem_reporter_manager_value`
  - `problem_reporter_department_value`
  - `problem_reporter_name`
  - `problem_reporter_roles`
```

Reply:
Can you make the same update to the similar entries in this CHANGELOG that list out field additions/removals.

Reply:
Updated the format!